src/Cron_Event_Command.php

```diff
@@ -233,6 +236,20 @@ public function run( $args, $assoc_args ) {
 			WP_CLI::error( $events );
 		}
 
+		$exclude = Utils\get_flag_value( $assoc_args, 'exclude' );
+
+		if ( ! empty( $exclude ) ) {
+			$exclude = explode( ',', $exclude );
+		}
+
+		$events = array_filter(
+			$events,
+			function ( $event ) use ( $args, $exclude ) {
+				return ( empty( $args ) || in_array( $event->hook, $args, true ) ) &&
+					( empty( $exclude ) || ! in_array( $event->hook, $exclude, true ) );
```

Comment:
This is pretty hard to read. Can you make it easier to read?

Reply:
Yes, Made the logic simpler.